docs/io/unified.rst

```diff
@@ -318,6 +327,78 @@ the keywords ``DATE``, ``DATE-*`` (ISO-8601 datetime strings) and the ``MJD-*``
 For more details regarding the FITS time paper and the implementation,
 refer to :ref:`fits_time_column`.
 
+Reading time columns in non-Astropy written FITS tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Reading FITS files which are compliant with the current FITS time standard,
+that is, the files written after the standard was set in stone, is supported
```

Comment:
Delete the whole "that is, the files .. set in stone," clause.  Many (most?) FITS files are still not compliant with that standard.  This work you have done may change that!